models/migrations/migrations.go

```diff
@@ -791,8 +792,18 @@ func dropTableColumns(sess *xorm.Session, tableName string, columnNames ...strin
 		}
 		tableSQL := string(res[0]["sql"])
 
+		// Get the index after the column definitions.
+		endColumnsIndex := strings.Index(tableSQL, "(")
+
+		// If endColumnsIndex is -1, which shouldn't be passed
+		// into a slice. We should return a error, as that also
+		// means that the substring wasn't found.
+		if endColumnsIndex == -1 {
+			return errors.New("Couldn't get index after column defintions")
```

Comment:
Maybe we do not need 4 line comments, reader can understand the logic, it's clear enough 😊

About Go error format: https://github.com/golang/go/wiki/CodeReviewComments#error-strings

```
Error strings should not be capitalized (unless beginning with proper nouns or acronyms) or end with punctuation, since they are usually printed following other context. That is, use fmt.Errorf("something bad") not fmt.Errorf("Something bad"), so that log.Printf("Reading %s: %v", filename, err) formats without a spurious capital letter mid-message. This does not apply to logging, which is implicitly line-oriented and not combined inside other messages.
```

Reply:
```
	// Get the index after the column definitions.
	endColumnsIndex := strings.Index(tableSQL, "(")
```

If I understand correctly, the purpose the `Index` is to find the beginning of the column definitions. For example:

`CREATE table_name (.....) ;`

Reply:
> If I understand correctly, the purpose the `Index` is to find the beginning of the column definitions. For example:

That should be indeed the purpose.